Lombiq.Walkthroughs/Assets/Scripts/walkthroughs.js

```diff
@@ -0,0 +1,3770 @@
+                            {
+                                action: function () {
+                                    goToRelativePage(
+                                        Shepherd.activeTour.options.id,
+                                        'audit_trail_intro',
+                                        'Admin',
+                                        'Admin');
+                                },
+                                classes: 'shepherd-button-secondary',
+                                text: 'Back',
+                            },
+                            nextButton,
+                        ],
+                        id: 'audit_trail_settings_page',
+                    },
+                    {
+                        title: 'Audit Trail',
+                        text: 'Click here to see the trimming settings.',
+                        attachTo: { element: 'a[href="#tab-trimming"]', on: 'top' },
+                        buttons: [
+                            backButton,
+                        ],
+                        id: 'audit_trail_trimming_tab_button',
+                        advanceOn: { selector: 'a[href="#tab-trimming"]', event: 'click' },
+                    },
+                    {
+                        title: 'Audit Trail',
+                        text: 'Here you can see the trimming settings.',
+                        attachTo: { element: '.edit-item', on: 'top' },
+                        canClickTarget: false,
+                        buttons: [
+                            {
+                                action: function () {
+                                    goToRelativePage(
+                                        Shepherd.activeTour.options.id,
+                                        'audit_trail_settings_page',
+                                        'Admin',
+                                        'Admin/Settings/AuditTrail');
+                                },
+                                classes: 'shepherd-button-secondary',
+                                text: 'Back',
+                            },
+                            nextButton,
+                        ],
+                        id: 'audit_trail_trimming_tab',
+                    },
+                    {
+                        title: 'Audit Trail',
+                        text: 'Click here to see the types of content whose events are recorded',
+                        attachTo: { element: 'a[href="#tab-content"]', on: 'top' },
+                        buttons: [
+                            backButton,
+                        ],
+                        id: 'audit_trail_content_tab_button',
+                        advanceOn: { selector: 'a[href="#tab-content"]', event: 'click' },
+                    },
+                    {
+                        title: 'Audit Trail',
+                        text: 'Click here to see the types of content whose events are recorded',
```

Comment:
Are you sure this is the right text here?
![image](https://github.com/Lombiq/Orchard-Walkthroughs/assets/19938236/b9bbb1c7-c013-4e07-9760-5a71f85013cd)

Reply:
![no-sonny](https://github.com/Lombiq/Orchard-Walkthroughs/assets/80963259/25a7dc03-f365-42c1-b908-125b91e8e2b6)